src/main/java/com/example/bookstore/model/Book.java

```diff
@@ -30,4 +34,6 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    @Column(name = "is_deleted", nullable = false)
```

Comment:
Hibernate creates column automatically in snake case format
```suggestion
    @Column(nullable = false)
```